DiscogsAPI/Extensions/DGCompany+DiscogsAPI.h

```diff
@@ -0,0 +1,22 @@
+//
+//  DGCompany+DiscogsAPI.h
+//  Pods
+//
+//  Created by Nate Rivard on 9/12/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+/**
+ Primitive values for DGCompany.entityType
+ */
+typedef NS_ENUM(NSInteger, DGCompanyType) {
```

Comment:
You dont need an extension file to declare this enum, it should be declared in `DGCompany.h`.
The `Extensions` group and subspec can then be removed.